modules/tinymce/src/core/main/ts/delete/CefDelete.ts

```diff
@@ -78,6 +80,11 @@ const backspaceDeleteRange = (editor: Editor, forward: boolean): Optional<() =>
     );
   }
 
+  if (isCefAtEdgeSelected(editor)) {
+    return Optional.some(() => {
+      deleteRng(editor.dom, editor.selection.getRng(), (elm) => elm === editor.getBody());
```

Comment:
This is only intended to be used for `textpattern` as it removes all content and doesn't pad, since the text pattern will replace it (which is also why it's in the `textpatterns` directory). This isn't a viable option here and we need to use something like what's used in `TableDelete` (see `deleteCellContents`) which will both clean up and pad the empty element. A good test case here would be something like a CEF element inside a list item.

Anyways, I'd suggest we move and rename that logic from `TableDelete` to a more shared space and that should be able to be used here I believe.